### PR TITLE
docs: Adding a note about GitHub Enterprise Cloud PAT requirement

### PR DESCRIPTION
**Pull request by ibuziuk**
## What does this pull request change?
docs: Adding a note about GitHub Enterprise Cloud PAT requirement

image

## What issues does this pull request fix or reference?
Related to https://issues.redhat.com/browse/CRW-4551

## Pull Request checklist

The author and the reviewers validate the content of this pull request with the following checklist, in addition to the [automated tests](code_review_checklist.adoc).

- Any procedure:
  - [ ] Successfully tested.
- Any page or link rename:
  - [ ] The page contains a redirection for the previous URL.
  - Propagate the URL change in:
    - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-frontend/src/services/bootstrap/branding.constant.ts)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/main/src/constants.ts)
- [ ] Builds on [Eclipse Che hosted by Red Hat](https://workspaces.openshift.com).
- [ ] the *`Validate language on files added or modified`* step reports no vale warnings.

### CODE REVIEW

**Review by l0rd — APPROVED** [review]

**Comment by ibuziuk** [comment]
@deerskindoll @vinokurig could you please review?

**Review by vinokurig — APPROVED** [review]

File: modules/end-user-guide/pages/using-a-git-provider-access-token.adoc [hunk 1]
```diff
@@ -41,6 +41,10 @@ On OpenShift, you can use the `oc` command-line tool to log in to the cluster:
 
 . Generate your access token on your Git provider's website.
 
++
+IMPORTANT: for GitHub Enterprise Cloud verify that the token is link:https://docs.github.com/en/enterprise-cloud@latest/authentication/authenticating-with-saml-single-sign-on/authorizing-a-personal-access-token-for-use-with-saml-single-sign-on[authorized for use within the organization].
```
Comment (deerskindoll) [inline modules/end-user-guide/pages/using-a-git-provider-access-token.adoc]:
```suggestion
+
[IMPORTANT]
====
For GitHub Enterprise Cloud, verify that the token is link:https://docs.github.com/en/enterprise-cloud@latest/authentication/authenticating-with-saml-single-sign-on/authorizing-a-personal-access-token-for-use-with-saml-single-sign-on[authorized for use within the organization].
====
```

**Review by deerskindoll — COMMENTED** [review]
changed formatting of the admonition, otherwise lgtm

**Comment by ibuziuk** [comment]
@deerskindoll applied, please approve and we can proceed with merging

**Review by deerskindoll — APPROVED** [review]
lgtm

**Pull request merged**